src/status_im/ui/screens/hardwallet/components.cljs

```diff
@@ -49,7 +49,7 @@
                                    :number    4
                                    :next-step :recovery-phrase}
    :card-ready                    {:label     :t/pairing
-                                   :number    5
+                                   :number    4
```

Comment:
just wondering what is this number for?

Reply:
this just a number to display along with step name in topbar component:

<img width="227" alt="screenshot 2019-02-27 19 47 18" src="https://user-images.githubusercontent.com/23836/53507199-81f99b00-3ac8-11e9-9bc8-d720a3197ed7.png">